packages/tools/devtools/devtools-browser-extension/src/popup/PopupScript.ts

```diff
@@ -28,4 +29,15 @@ browser.tabs.query({ active: true, currentWindow: true }, (tab) => {
 		'To use the Fluid Devtools, open the browser Devtools pane (F12) and click the "Fluid Developer Tools" tab.';
 
 	document.body.append(popupElement);
+	initializePopupView(popupElement).then(() => {
+		if (tabs.length === 0 || tabs[0].id === undefined) {
+			console.error("No active tab found or tab ID is undefined");
+			return;
+		}
+		// Send message to webpage
+		port.postMessage({ tabID: tabs[0].id, data: "Requesting supportedFeatures" });
```

Comment:
This confuses me a bit... we're asking the background script (the thing that we can talk to through `port` for supported features. Do we have that data there? I thought content script is the one that we would need to ask.

Once we have the tab id above, shouldn't we be doing something like [`tabs.sendMessage(<tabId>, <message>)`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/sendMessage), to talk directly to the content script and ask for supported features?